hw/top_earlgrey/data/ip/chip_clkmgr_testplan.hjson

```diff
@@ -192,7 +192,7 @@
             SiVal: This is a connectivity test only, not suitable for sival.
             '''
       stage: V2
-      si_stage: NA
+      si_stage: SV3
```

Comment:
The testpoint describes this test as a connectivity only, and the code in clkmgr_jitter_test.c is pretty lame, and doesn't take into account the jitter_regwen bit. In fact, I'd say that test is not useful. Instead, jitter can be tested via the testpoint chip_sw_clkmgr_jitter_cycle_measurements, and even that test doesn't take jitter_regwen into account.

The point about jitter_regwen is that it may not be possible to change the state of jitter_enable, so it may be frozen to either enable or disable: if enable we will never change and the frequencies will always correspond to the nominal frequency; if disabled we cannot switch it, and only then will the frequency measurements really be able to confirm it. This area seems weak: I filed https://github.com/lowRISC/opentitan/issues/25177 regarding these tests.

Reply:
Reverted to NA